src/algebra/scalar/scalar.rs

```diff
@@ -147,16 +147,29 @@ impl<'de, C: CurveGroup> Deserialize<'de> for Scalar<C> {
     }
 }
 
+/// A type alias for a result that resolves to a `Scalar`
+pub type ScalarResult<C> = ResultHandle<C, Scalar<C>>;
+/// A type alias for a result that resolves to a batch of `Scalar`s
+pub type BatchScalarResult<C> = ResultHandle<C, Vec<Scalar<C>>>;
+
+impl<C: CurveGroup> ScalarResult<C> {
+    /// Exponentiation
+    pub fn pow(&self, exp: usize) -> Self {
```

Comment:
don't have a strong opinion here but seems a lil funky to have the exponent be a usize, why not just make it a `u64`?

Reply:
sure